thirdparty/github.go

```diff
@@ -36,6 +36,32 @@ const (
 	GithubInvestigation = "Github API Limit Investigation"
 )
 
+var UnblockedGithubStatuses = []string{
+	githubPrBehind,
+	githubPrClean,
+	githubPrDirty,
+	githubPrDraft,
+	githubPrHas_Hooks,
+	githubPrUnknown,
+	githubPrUnstable,
+}
+
+const (
+	// all PR statuses except for "blocked" based on statuses listed here:
```

Comment:
Why is that we aren't just checking if it == blocked?

Reply:
Because the field is not documented so it can change without notice. 

If we let through anything that is not blocked, we risk it failing silently without us knowing if the field changes. With this setup, it needs to be one of the fields listed to go through, so we are less likely to merge something that shouldn't be merged.

Reply:
can we add this rational to the function comment?

Reply:
done